### PR TITLE
logging build id and run id

### LINKED ISSUE #5562: Stress Tests - Include runner runId and pipeline buildId on logs
## Work Item
**Describe the outcome you expect**
We need to be able to go back and forth between logs about the stress test and the pipeline run.

**Approach**
Use ChildLogger to add runId and buildId to the logs

### PR DESCRIPTION
**Pull request by chensixx**
Fixes #5562
Added an env variable for build id (shown when pipeline is run in ADO) 
Added a column that will show the build id , and another column showing the run id

edit [markfields] - I updated the issue number since I broke out this work into its own issue]

### CODE REVIEW

File: packages/test/service-load-test/src/nodeStressTest.ts [hunk 1]
```diff
@@ -32,13 +33,14 @@ const codeDetails: IFluidCodeDetails = {
...
 
-function createLoader(testDriver: ITestDriver) {
+function createLoader(testDriver: ITestDriver, runId?: number) {
     // Construct the loader
     const loader = new Loader({
         urlResolver: testDriver.createUrlResolver(),
         documentServiceFactory: testDriver.createDocumentServiceFactory(),
         codeLoader,
-        logger,
+        logger: ChildLogger.create(logger, undefined, {all: {buildId: process.env.FLUID_PIPELINE_BUILD_ID,
```
Comment (anthony-murphy) [inline packages/test/service-load-test/src/nodeStressTest.ts:42]: you don't need to add a new variable. see https://docs.microsoft.com/en-us/azure/devops/pipelines/process/variables?view=azure-devops&tabs=yaml%2Cbatch#system-variables

File: packages/test/service-load-test/src/nodeStressTest.ts [hunk 2]
```diff
@@ -32,13 +33,14 @@ const codeDetails: IFluidCodeDetails = {
 
 const codeLoader = new LocalCodeLoader([[codeDetails, fluidExport]]);
 
-function createLoader(testDriver: ITestDriver) {
+function createLoader(testDriver: ITestDriver, runId?: number) {
```
Comment (anthony-murphy) [inline packages/test/service-load-test/src/nodeStressTest.ts:36]: what is runId? i don't see it used anywhere

File: packages/test/service-load-test/src/nodeStressTest.ts [hunk 3]
```diff
@@ -32,13 +33,14 @@ const codeDetails: IFluidCodeDetails = {
...
     // Construct the loader
     const loader = new Loader({
         urlResolver: testDriver.createUrlResolver(),
         documentServiceFactory: testDriver.createDocumentServiceFactory(),
         codeLoader,
-        logger,
+        logger: ChildLogger.create(logger, undefined, {all: {buildId: process.env.BUILD_BUILDID,
```
Comment (anthony-murphy) [inline packages/test/service-load-test/src/nodeStressTest.ts:42]: we can check this in, but i really think this should just be confgured in the logger we inject by default

File: packages/test/service-load-test/src/nodeStressTest.ts (hunk 2 above)
Comment (anthony-murphy) [inline packages/test/service-load-test/src/nodeStressTest.ts:36]: i don't think runId should be optional

Comment (markfields) [reply packages/test/service-load-test/src/nodeStressTest.ts:36]: It's also called from `initialize` in the orchestrator process which has no runId.

File: packages/test/service-load-test/src/nodeStressTest.ts (hunk 3 above)
Comment (markfields) [reply packages/test/service-load-test/src/nodeStressTest.ts:42]: Good idea, added a task over here:  https://dev.azure.com/fluidframework/internal/_backlogs/backlog/internal%20Team/Issues/?workitem=9

There's a few clean-up items in that backlog, I'll open a GitHub issue for April to cover those.

Comment (markfields) [reply packages/test/service-load-test/src/nodeStressTest.ts:42]: https://github.com/microsoft/FluidFramework/issues/5567

**Review by anthony-murphy — APPROVED** [review]

**Review by markfields — APPROVED** [review]
🎂 LGTM

**Pull request merged**